tests/tutor/test_tutor_teacher.py

```diff
@@ -0,0 +1,1330 @@
+"""Test the Tutor teacher functions."""
+
+from tests.markers import nondestructive, test_case, tutor
```

Comment:
Flake8 - `nondestructive` isn't used -- Are some of these tests non-destructive?